rosbag2_storage_mcap/test/rosbag2_storage_mcap/test_mcap_storage.cpp

```diff
@@ -36,6 +36,12 @@ bool operator==(const TopicInformation & lhs, const TopicInformation & rhs)
 {
   return lhs.topic_metadata == rhs.topic_metadata && lhs.message_count == rhs.message_count;
 }
+
+bool operator==(const MessageDefinition & a, const MessageDefinition & b)
+{
+  return a.topic_type == b.topic_type && a.encoding == b.encoding &&
+         a.encoded_message_definition == b.encoded_message_definition;
+}
```

Comment:
@james-rms Could you please move this comparison operator inside `MessageDefinition` declaration?
I have a suspicious that it will be required from another places as well. At least from test for SQlite3 plugin.